### ISSUE TITLE
Using pure JavaScript with cocoon events

### ISSUE THREAD

**Issue by Colopatiron** [nathanvda/cocoon#525]
I would like to use pure JS instead of using jquery to implement callbacks. I tried doing something like

```javascript
var travellerProfiles = document.querySelector('#traveller-profiles')
travellerProfiles.addEventListener('cocoon:after-insert', function () {
  console.log('Added something.')
})
```

but it doesn't work, whereas using jquery constructs (like those from the examples) do. Is there anything I'm doing wrong and is there any way to make it work?

**Comment by nathanvda** [comment]
Hi, apparently this is not possible. I was not aware of this, but when triggering custom events using jquery they are not capturable using pure javascript. 

I found a bug report here: https://bugs.jquery.com/ticket/11047 which kind of explains why this is and why they will not change this. 

I also found a possible way to trigger a custom event that then could be captured using either pure javascript or jquery, but in our code why encapsulate the to-be-inserted or inserted partial in a `$()` wrapper for ease of use.

So at the moment jquery is a hard dependency and for me i do not see a benefit in dropping it. 

Also see https://github.com/nathanvda/cocoon/issues/441
